ts/features/bonus/bpd/screens/optInPaymentMethods/OptInPaymentMethodsCashbackUpdate.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { SafeAreaView, ScrollView } from "react-native";
+import { View } from "native-base";
+import I18n from "../../../../../i18n";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../../components/core/typography/H1";
+import ButtonDefaultOpacity from "../../../../../components/ButtonDefaultOpacity";
+
+const OptInPaymentMethodsCashbackUpdate = () => (
```

Comment:
```suggestion
const OptInPaymentMethodsCashbackUpdateScreen = () => (
```
Since it is not a component what do you think to add `Screen` suffix?

Reply:
Sure, renamed here [f4034c5](https://github.com/pagopa/io-app/pull/3701/commits/f4034c5af8c9236546822023f001bbd373e4e301)